app/views/assignments/show.html.erb

```diff
@@ -0,0 +1,43 @@
+<% content_for :pre_content_nav do %>
+  <% render partial: "shared/back_link", locals: {href: root_path} %>
+<% end %>
+
+<%= render partial: "projects/index/project_summary", locals: {project: @project} %>
```

Comment:
I don't think this is the correct summary - this is the one used on index pages.

Reply:
Oops, too many partials 🤦

Reply:
for sure!